aodncore/vocab/platform_code_vocab.py

```diff
@@ -17,8 +17,7 @@
 import warnings
 import xml.etree.ElementTree as ET
 
-from six.moves.urllib_request import urlopen
-
+from urllib2 import urlopen
```

Comment:
This should be left how it was, since the six.moves is to support Python 2 and 3, and will help when we need to look at that more seriously very soon!

Reply:
Since we're almost about to embark on a Python 3 migration, we really need to import using the six namespace during that transition